[2020-12-08 23:40:28,967] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Airflow_task.Data_from_table2 2020-12-08T18:05:00+00:00 [queued]>
[2020-12-08 23:40:28,983] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Airflow_task.Data_from_table2 2020-12-08T18:05:00+00:00 [queued]>
[2020-12-08 23:40:28,983] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2020-12-08 23:40:28,983] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2020-12-08 23:40:28,984] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2020-12-08 23:40:28,993] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): Data_from_table2> on 2020-12-08T18:05:00+00:00
[2020-12-08 23:40:28,997] {standard_task_runner.py:54} INFO - Started process 21635 to run task
[2020-12-08 23:40:29,045] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'Airflow_task', 'Data_from_table2', '2020-12-08T18:05:00+00:00', '--job_id', '306', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/server-elastic.py', '--cfg_path', '/var/folders/x0/npyjyyg15zj3krdkh0pv612w0000gn/T/tmpgqctky5y']
[2020-12-08 23:40:29,047] {standard_task_runner.py:78} INFO - Job 306: Subtask Data_from_table2
[2020-12-08 23:40:29,086] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: Airflow_task.Data_from_table2 2020-12-08T18:05:00+00:00 [running]> 153.225.168.192.in-addr.arpa
[2020-12-08 23:40:35,196] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2020-12-08 23:40:35,196] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2020-12-08 23:40:35,197] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 61. Disconnecting.
[2020-12-08 23:40:35,198] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-08 23:40:35,245] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2020-12-08 23:40:35,245] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 61. Disconnecting.
[2020-12-08 23:40:35,246] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-08 23:40:35,295] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2020-12-08 23:40:35,295] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 61. Disconnecting.
[2020-12-08 23:40:35,295] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-08 23:40:35,296] {logging_mixin.py:112} INFO - Error::::: NoBrokersAvailable
[2020-12-08 23:40:35,296] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-12-08 23:40:35,300] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=Airflow_task, task_id=Data_from_table2, execution_date=20201208T180500, start_date=20201208T181028, end_date=20201208T181035
[2020-12-08 23:40:38,975] {local_task_job.py:102} INFO - Task exited with return code 0
